server/src/main/xsd/gateway-config-201409.xsd

```diff
@@ -1253,6 +1253,18 @@
                     </documentation>
                 </annotation>
             </element>
+            <element name="http.keepalive.max.connections" maxOccurs="1" minOccurs="0" type="nonNegativeInteger" default="2">
```

Comment:
The annotation is not clear. It would be better if you took the perspective of predicting and answering questions such as these: When would I use this? Why? Add recommendations/best practices for setting size (small enough?).  

Also terminology is a problem. What is an upstream server -- is that one of the Gateways (DMZ or internal/Enterprise)? What is the "worker"?